bindings/node/native/src/classes/synced_account/mod.rs

```diff
@@ -17,15 +13,19 @@ mod repost;
 mod send;
 
 #[derive(Clone)]
-pub struct SyncedAccountWrapper(Arc<RwLock<SyncedAccount>>, String);
+pub struct SyncedAccountWrapper(pub String);
```

Comment:
again, more a matter of taste than an issue. This struct is `pub` so I guess it gets exported and is part of the API of the library. In that case I would name that type `SyncedAccount`, and the type that it wraps `SyncedAccountInner`. That makes the API nice to read IMHO.

Reply:
this is just a hidden type on the JavaScript class that the binding generates for the synced account so I think it's fine to leave it as is; it's only public because the JS-generated code is also public.